scripts/update-istio-dependencies.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+[ -z "${1:-}" ] && { echo "Usage: $0 <version>"; exit 1; }
+
+version=$1
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+build_dir=${script_dir}/../build
+
+${script_dir}/remove-istio-dependencies.sh ${build_dir}
+
+pushd ${build_dir}
+
+echo "clone istio api repository"
+git clone -q https://github.com/istio/api
+pushd api
+git reset --hard ${version} >/dev/null
+popd
```

Comment:
```suggestion
git clone -q -b ${version} https://github.com/istio/api
```

It's not exactly the same, but I think this version is less confusing. `git reset --hard 1.9.3` moves the local `master` branch to the 1.9.3 tag, which is not exactly clean. My version simply switches to the commit where 1.9.3 points to, without using a local branch.